app/assets/stylesheets/fae/globals/imports/finescss/_fine_functions.scss

```diff
@@ -57,3 +57,19 @@
     @return #00f;
   }
 }
+
+/// Choose from a light or dark contrasting foreground color given a background
```

Comment:
Original function for reference: https://gist.github.com/mkel23/0d8d783b91be69047376#file-_yiq-scss